monitor-iot-data-with-freeboard/README.md

```diff
@@ -0,0 +1,210 @@
+# How to create a realtime IoT dashboard for Kuzzle
+
+## Table of Content
+
+<!-- TOC -->
+
+- [How to create a realtime IoT dashboard for Kuzzle](#how-to-create-a-realtime-iot-dashboard-for-kuzzle)
+    - [Table of Content](#table-of-content)
+    - [Purpose](#purpose)
+    - [Prerequisite](#prerequisite)
+    - [Howto steps](#howto-steps)
+        - [Step 0: Getting Kuzzle Ready](#step-0-getting-kuzzle-ready)
+        - [Step 1: Enable Kuzzle Datasource Plugin](#step-1-enable-kuzzle-datasource-plugin)
+        - [Step 2: Kuzzle Datasource Configuration](#step-2-kuzzle-datasource-configuration)
+        - [Step 3: Adding datasource for the SIN_GENERATOR device](#step-3-adding-datasource-for-the-sin_generator-device)
+        - [Step 4: Adding a widget to visualize data from SIN_GENERATOR device](#step-4-adding-a-widget-to-visualize-data-from-sin_generator-device)
+        - [Step 5: Add other devices](#step-5-add-other-devices)
+    - [Conclusion](#conclusion)
+
+<!-- /TOC -->
+
+## Purpose
+
+The purpose of this howto is to learn how to create a realtime visualization dashboard for your [Kuzzle](http://www.kuzzle.io)-based by adding a simple plugin to [freeboard](http://freeboard.io), an open source dashboard.
+
```

Comment:
The purpose of this how-to is to learn how to create a real-time visualization for [Kuzzle](http://www.kuzzle.io) by adding a simple plugin to [freeboard](http://freeboard.io), an open source dashboard.